lfucg-exactions/client/js/components/Pagination.js

```diff
@@ -2,30 +2,78 @@ import React from 'react';
 import { getPagination } from '../actions/apiActions';
+import { formUpdate } from '../actions/formActions';
 
 class Pagination extends React.Component {
 
     render() {
-        if (window.pageYOffset > 200) {
+        if (window.pageYOffset > 250) {
             window.scroll(0, 0);
         }
+        
         const {
             onPaginationChangePage,
             activeForm,
+            changePageSize,
         } = this.props;
 
         // relates to pagination size in backend
-        const paginationSize = 10;
+        const paginationSize = activeForm.page_size || 10;
 
         return (
-            <div className="row">
-                <div className="col-xs-12 text-center">
-                    <ul className="pagination">
-                        <li><button aria-label="previous" className="btn btn-default" disabled={!activeForm.prev || activeForm.query} onClick={() => onPaginationChangePage(activeForm.prev)}><i className="fa fa-angle-double-left" aria-hidden="true" /></button></li>
-                        <li>&nbsp;Page {activeForm.next ? (activeForm.next.charAt(activeForm.next.indexOf('=') + 1) - 1) : (Math.ceil(activeForm.count / paginationSize)) }&nbsp;</li>
-                        <li><button aria-label="next" className="btn btn-default" disabled={!activeForm.next || activeForm.query} onClick={() => onPaginationChangePage(activeForm.next)}><i className="fa fa-angle-double-right" aria-hidden="true" /></button></li>
-                    </ul>
+            <div className="container-fluid">
+                <div className="row">
+                    <div className="col-xs-12 text-center">
+                        <ul className="pagination">
+                            <li><button aria-label="previous" className="btn btn-default" disabled={!activeForm.prev} onClick={() => onPaginationChangePage(activeForm.prev)}><i className="fa fa-angle-double-left" aria-hidden="true" /></button></li>
+                            <li>&nbsp;Page {activeForm.next ? (activeForm.next.charAt(activeForm.next.indexOf('=') + 1) - 1) : (Math.ceil(activeForm.count / paginationSize)) }&nbsp;</li>
+                            <li><button aria-label="next" className="btn btn-default" disabled={!activeForm.next} onClick={() => onPaginationChangePage(activeForm.next)}><i className="fa fa-angle-double-right" aria-hidden="true" /></button></li>
+                        </ul>
+                    </div>
                 </div>
+                {activeForm.count > 10 &&
+                    <div>
+                        <div className="row">
+                            <div className="col-xs-12 col-sm-4 col-sm-offset-4 text-center">Results per page</div>
+                        </div>
+                        <div className="row">
+                            <div className="col-xs-12 col-sm-4 col-sm-offset-4 text-center">
+                                <button
+                                  aria-label="changePageSize"
```

Comment:
Make the aria-labels separate words so a screen reader will read them as words instead of letters.